src/index.html

```diff
@@ -8,6 +8,34 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <header class="header" data-qa="card">
```

Comment:
This is not a header
```suggestion
    <article class="card" data-qa="card">
```